### DISCUSSION TITLE
请问停止维护的西电研究生毕业论文版本里面关于每章的标题字体在哪个文件里改？我的绪论字体没有加粗。

### DISCUSSION THREAD

**Discussion by yzyzzzzz** [note286/xduts#181] [Q&A] [answered]
![1](https://github.com/note286/xduts/assets/90914433/d3ea5b12-4e87-4940-a6bb-12d0269a7a54)

**Comment by note286** [comment, marked as answer]
停止维护的意思就是没有技术支持。

**Reply by yzyzzzzz** [reply]
这个是我同学出现的问题，我用停止维护的并没有出现这样的问题。想问一下，停止维护就是没有技术支持但是那个模板本身没有问题 如果遇到问题自己解决，是这个意思吧？

**Reply by note286** [reply]
无论怎么样，你应该去 https://github.com/note286/xdupgthesis/issues 交流，而不是这里。

**Reply by yzyzzzzz** [reply]
好的，已提问。谢谢
